### PR TITLE
Small fixes tabs opening

### PR DESCRIPTION
**Pull request by AlexandreSi**

### CODE REVIEW

File: newIDE/app/src/MainFrame/EditorTabs/UseEditorTabsStateSaving.js [hunk 1]
```diff
@@ -155,8 +155,10 @@ const useEditorTabsStateSaving = ({
         );
       }
 
+      let openedEditorsCount = 0;
       for (const editorOpeningOption of editorsOpeningOptions) {
         newEditorTabs = openEditorTab(newEditorTabs, editorOpeningOption);
+        openedEditorsCount++;
```
Comment (D8H) [inline newIDE/app/src/MainFrame/EditorTabs/UseEditorTabsStateSaving.js:161]: Is `openedEditorsCount` the same as `editorsOpeningOptions.length`?

File: newIDE/app/src/MainFrame/index.js [hunk 2]
```diff
@@ -1983,10 +1983,12 @@ const MainFrame = (props: Props) => {
...
                 hasAPreviousSaveForEditorTabsState(currentProject)
               ) {
-                openEditorTabsFromPersistedState(currentProject);
+                const openedEditorsCount = openEditorTabsFromPersistedState(
+                  currentProject
+                );
                 setIsLoadingProject(false);
                 setLoaderModalProgress(null, null);
-                openProjectManager(false);
+                openProjectManager(openedEditorsCount === 0);
```
Comment (D8H) [inline newIDE/app/src/MainFrame/index.js:1991]: Should it also open the first scene?

File: newIDE/app/src/MainFrame/EditorTabs/UseEditorTabsStateSaving.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/EditorTabs/UseEditorTabsStateSaving.js:161]: Indeed, will change it

File: newIDE/app/src/MainFrame/index.js (hunk 2 above)
Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/index.js:1991]: You're right, it will be consistent with the opening of a new project with multiple scenes

**Review by ClementPasteau — APPROVED** [review]
lgtm

File: newIDE/app/src/MainFrame/EditorTabs/EditorTabsHandler.js [hunk 3]
```diff
@@ -157,7 +157,10 @@ export const openEditorTab = (
 
   return {
     ...state,
-    editors: [...state.editors, editorTab],
+    editors:
+      key === 'start page'
```
[the remark this answers is not in the document]
Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/EditorTabs/EditorTabsHandler.js:161]: I'll add a comment: it's to make sure the home page is always the first tab

**Pull request merged**